src/Hyperwallet/Model/BankAccount.php

```diff
@@ -784,11 +784,13 @@ public function getDateOfBirth() {
     /**
      * Set the date of birth
      *
-     * @param \DateTime|null $dateOfBirth
+     * @param string $dateOfBirth |null
      * @return BankAccount
      */
-    public function setDateOfBirth(\DateTime $dateOfBirth = null) {
-        $this->dateOfBirth = $dateOfBirth == null ? null : $dateOfBirth->format('Y-m-d');
+    public function setDateOfBirth($dateOfBirth = null)
+    {
+        $stringToDate = $dateOfBirth === null ? null : new \DateTime($dateOfBirth);
+        $this->dateOfBirth = $stringToDate == null ? null : $stringToDate->format('Y-m-d');
```

Comment:
As this date manipulation is done in several places, I suggest to make a common util method and use it in different places

Reply:
Done, thank you.